config/env/default.js

```diff
@@ -28,5 +28,5 @@ module.exports = {
   sessionKey: 'sessionId',
   sessionCollection: 'sessions',
   logo: 'modules/core/client/img/brand/logo.png',
-  favicon: 'modules/core/client/img/brand/favicon.ico'
+  favicon: './modules/core/client/img/brand/favicon.ico'
```

Comment:
@codydaig I'll look into this over the weekend.

My initial thought, is that the `./` isn't needed since `path.resolve` is being used. I'll confirm this when I test.